src/plugins/intel_cpu/src/nodes/kernels/registers_pool.hpp

```diff
@@ -119,6 +126,31 @@ class RegistersPool {
         }
     }
 
+    template<typename TReg>
+    inline TReg getInplaceFree(std::vector<Xbyak::Reg>& not_available) const {
```

Comment:
In my perception, RegisterPool should remain the bare minimum of the interface functionality in spite of simplicity, but the interface should be complete (as it is at the time it is merged). Such utility code should be placed in `jit_base`.

Reply:
Agree. But getInplaceFree() is no longer a part of this PR.